spark/src/main/scala/io/spark/ddf/etl/TransformationHandler.scala

```diff
@@ -22,7 +22,12 @@ import io.ddf.etl.{TransformationHandler ⇒ CoreTransformationHandler}
 import io.ddf.exception.DDFException
 import io.spark.ddf.SparkDDF
 
+import java.io.PrintStream
+import com.google.api.client.util.LoggingOutputStream
+
 class TransformationHandler(mDDF: DDF) extends CoreTransformationHandler(mDDF) {
+  // Rsession to use
+  val rsess = Rsession.newInstanceTry(new PrintStream(new LoggingOutputStream(mLog, false), true), null)
```

Comment:
@dabaitu this generally looks good! It's seems RSession sensibly re-use most of the Rserve API e.g. REXP so the changes are not too bad.

What is really important is to understand the inter-process overall data flow to execute R code on top of DDF. See this doc: https://docs.google.com/a/adatao.com/document/d/1BOl-FU-jKijlMGUaYy4gC0SIo0KAYwr3-o_G3mdd_uo/edit

In this case, each `rsess` should be created inside the dfrdd.map so that each data partition gets a new R instance and the map tasks can proceed in parallel. (BTW, It's a map but it's really a mapPartition behind the scene, `partdf` is a whole partition represented as a REXP).

Re logging, this is a good question. When we start Rserve manually we redirect it to a log file on each slave nodes. Again, note that when you create the Rsession you are sitting at a slave node and presented a data partition to work on. 

@mbbui to suggest something sensible re logging. Redirect to stdout is nice if YARN is able to capture them from the slave executors.